seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -105,7 +105,8 @@ void beforeClass() throws Exception {
         node2Tags.setAttribute("node", "node2");
         Config node2hzconfig = node1Config.getHazelcastConfig().setMemberAttributeConfig(node2Tags);
         node2Config = ConfigProvider.locateAndGetSeaTunnelConfig();
-        node2Config.getEngineConfig().getHttpConfig().setPort(8081);
+        // Dynamically generated port
+        node2Config.getEngineConfig().getHttpConfig().setEnableDynamicPort(true);
```

Comment:
@Hisoka-X Already verified this by not specifying the port

Reply:
> @Hisoka-X Already verified this by not specifying the port

Can we provide a port range? Otherwise, how do I know which port is bound?

Reply:
> Can we provide a port range? Otherwise, how do I know which port is bound?

Get, I will add a configuration control to the cumulative range, like 'port-count', Res: 8080-8180, What do you think

Reply:
> > Can we provide a port range? Otherwise, how do I know which port is bound?
> 
> Get, I will add a configuration control to the cumulative range, like 'port-count', Res: 8080-8180, What do you think @liugddx

I think it's ok